docs/deploying.md

```diff
@@ -167,15 +166,24 @@ Resources:
       LaunchTemplateData:
         ImageId: !Ref FckNatAMIParameter
         InstanceType: t4g.nano
+        NetworkInterfaces:
+          - DeviceIndex: 0
+            AssociatePublicIpAddress: true
+            Groups:
+            - !GetAtt [FckNatSecurityGroup, GroupId]
```

Comment:
While this is more explicit, if you are deploying your NAT into a public subnet, associating a public IP address is the default.

Reply:
I believe by default, and EC2 instance launched in a public subnet is not assigned a public IP.

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-subnet.html#cfn-ec2-subnet-mappubliciponlaunch

This is just a problem I ran into, and I didn't want to go changing my VPC defaults just to launch a NAT instance